mkosi/distributions/ubuntu.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+from typing import Set
+
+from ..backend import MkosiState, add_packages
+from .debian import DebianInstaller
+
+
+class UbuntuInstaller(DebianInstaller):
+    repositories_for_boot = {"universe"}
+
+    @classmethod
+    def add_default_kernel_package(cls, state: MkosiState, extra_packages: Set[str]) -> None:
```

Comment:
We can easily do this calculation before calling a shared install function and pass the requested default kernel package as an argument.